packages/eds-core-react/src/components/Breadcrumbs/Breadcrumb.tsx

```diff
@@ -24,30 +25,47 @@ const StyledTypography = styled(Typography)<StyledProps>`
   color: ${typography.color};
   ${({ maxWidth }) => css({ maxWidth })}
 `
+type OverridableSubComponent = OverridableComponent<
+  BreadcrumbProps,
+  HTMLAnchorElement
+> & {
+  displayName?: string
+}
 
 export type BreadcrumbProps = {
   /* Max label width in pixels,
    * truncate long labels based on this width */
   maxWidth?: number
   /** Children is breadcrumb text */
   children: string
+  /** Override element type */
+  as?: ElementType
 } & React.AnchorHTMLAttributes<HTMLAnchorElement>
 
-export const Breadcrumb = forwardRef<HTMLAnchorElement, BreadcrumbProps>(
-  function Breadcrumb({ children, maxWidth, href, ...other }, ref) {
+export const Breadcrumb: OverridableSubComponent = forwardRef(
+  function Breadcrumb({ children, maxWidth, href, as, ...other }, ref) {
     const props = {
       ...other,
       href,
       ref,
       maxWidth,
     }
+
     const showTooltip = maxWidth > 0
     const isHrefDefined = href !== undefined
+    let forwardedAs: ElementType
```

Comment:
Could this be rewritten as a ternarery operator? forwaredAs could be initiated with `as` value and then ternarery for if `isHrefDefined` or not. Maybe wrap it in a useCallback also?